debug.js

```diff
@@ -38,7 +38,7 @@ var prevColor = 0;
  * Previous log timestamp.
  */
 
-var prevTime;
+var prevTime = {};
```

Comment:
In this case, the previous time should be held inside of the debug closure.  That way the value is encapsulated by the log instance.  Technically, this line is a leak